pkg/chunked/dump/dump.go

```diff
@@ -218,7 +233,7 @@ func GenerateDump(tocI interface{}, verityDigests map[string]string) (io.Reader,
 				Mode: 0o755,
```

Comment:
Isn’t this special case for `/` now redundant with the new logic?

Reply:
I could drop the condition `|| (sanitizeName(toc.Entries[0].Name) != "/")` but we still need it for `len(toc.Entries) == 0` as we never call `dumpNode()`

Reply:
Oh, I didn’t realize that was also a practical possibility.

I do think removing the second condition would be valuable in itself — and it would make the “empty entries” special case a bit more noticeable.